presto-main/src/test/java/io/prestosql/sql/planner/TestPlanMatchingFramework.java

```diff
@@ -140,13 +143,13 @@ public void testTableScan()
     @Test
     public void testJoinMatcher()
     {
-        assertPlan("SELECT o.orderkey FROM orders o, lineitem l WHERE l.orderkey = o.orderkey",
+        assertPlan("SELECT o.orderkey FROM orders o, lineitem l WHERE l.orderkey = o.orderkey", disableCbo(),
```

Comment:
Please format this this way:

```suggestion
        assertPlan(
        "SELECT o.orderkey FROM orders o, lineitem l WHERE l.orderkey = o.orderkey",
         disableCbo(),
```